README.md

```diff
@@ -139,13 +153,29 @@ Apache License 2.0, see LICENSE.txt for more information.
 
 # Under the hood
 
-`Virtualbox` is a utility that lets people creating virtual machines on your computer.
+`Virtualbox` is a utility that lets people creating virtual machines (VM) on your computer. The created VM is the GUEST. The computer you run Virtualbox on is the HOST
 
 `Vagrant` is used as a tool to define VMs characteristics, that will be then run through Virtualbox - in other words, it is used so we can write a manifesto that defines the characteristics of a VM, and the VM generated has always the same characteristics. It is useful in this case to model the development VM just as if it was the server on which we will run the application.
 
-`Make` is a tool that, among other things, chains commands together. So, for instance, you write in the `Makefile` that `a` runs a specific long command, `b` a different long command, and you can call the commands with `make a` or `make b`. You can also write a command `c` which is a chain of `a` followed by `b`. We use it to set a 'flow' of operations that should be followed (e.g. as explained, `make bootstrap` chains 3 operations, and one such operation is used very often i.e. `build` and/or `start`)
+`Make` is a tool that, among other things, chains commands together. So, for instance, you write in the `Makefile` that `a` runs a specific long command, `b` a different long command, and you can call the commands with `make a` or `make b`. You can also write a command `c` which is a chain of `a` followed by `b`. We use it to set a 'flow' of operations that should be followed (e.g. as explained, `make bootstrap` chains 3 operations, and one such operation is used very often i.e. `build` and/or `start`). This is used on the _GUEST_.
```

Comment:
```suggestion
`Make` is a tool that, among other things, chains commands together, automatically handling dependencies between those commands. So, for instance, you write in the `Makefile` that `a` runs a specific long command, `b` a different long command, and you can call the commands with `make a` or `make b`. You can also write a command `c` which is a chain of `a` followed by `b`. We use it to set a 'flow' of operations that should be followed (e.g. as explained, `make bootstrap` chains 3 operations, and one such operation is used very often i.e. `build` and/or `start`). This is used on the _GUEST_.
```
Is the makefile indented to be only ever executed on the GUEST? Then I'd like a command stating so on the top of the `Makefile` file. "Those commands are intended to be executed on the GUEST system, i.e. within the vagrant VM".

I think `make bootstrap` is intended to be executed on the HOST?

Reply:
no, `make bootstrap` is executed *by* vagrant on the GUEST (so, one never even notices)

Added your suggestion

Reply:
no to your second question, yes to your first question